workflow/rules/common.smk

```diff
@@ -544,11 +540,17 @@ def get_filter_targets(wildcards, input):
         return ""
 
 
+def get_annotation_filter(wildcards):
+    filter = config["calling"]["fdr-control"]["events"][wildcards.event]["filter"]
```

Comment:
Allow filter to be just a string here (for convenience and compatibility).

Reply:
This should work now.